src/lib/create-utils.js

```diff
@@ -58,6 +64,34 @@ exports.getTemplateDefinition = async (maybeTemplateName, manifestPromise) => {
     return executePrompts(manifest);
 };
 
+/**
+ * Fetch local readme suffix from the manifest and append it to the readme.
+ * @param {string} readmePath
+ * @param {Promise<object>} manifestPromise
+ */
+exports.enhanceReadmeWithLocalSuffix = async (readmePath, manifestPromise) => {
+    const manifest = await manifestPromise;
+    // If the fetch failed earlier, the resolve value of
+    // the promise will be the error from fetching the manifest.
+    if (manifest instanceof Error) throw manifest;
+
+    try {
+        let localReadmeSuffix = '';
```

Comment:
I would use stream if there is an option.
```
const suffixStream = await this.httpsGet(manifest.localReadmeSuffixUrl);
const readmeStream = fs.createWriteStream(readmePath, { flags: 'a' });
readmeStream.write('\n\n');
await promisify(pipeline)(suffixStream, readmeStream);
```

Reply:
Thx looks nicer too :)